src/index.js

```diff
@@ -0,0 +1,33 @@
+import navBarComponent from './navBarElement';
+import contentHomeComponent from './home';
+import contentMenuComponent from './menu';
+import contentContactComponent from './contact';
+
+console.log();
```

Comment:
No console.log commited to repo. It's ugly, it kills performance. and it can make confidential data be visible in the browser tools.